packages/design-tokens/sass/border.scss

```diff
@@ -57,12 +51,6 @@ $border-borderless-border-color-rgb: var(
   0,
   0
 );
-$border-borderless-border-color-rgb-params: var(
-  --border-borderless-border-color-rgb,
-  0,
-  0,
-  0
-);
```

Comment:
@ActuallyACat made another small improvement: to not generate `-rgb-params` for the new generation of token names.